### PR TITLE
Korriban tileset

### PR DESCRIPTION
**Pull request by Cavcode**
PhoD Terria Desert tileset
Booba not included.
image

### CODE REVIEW

**Comment by zunath** [comment]
Compile the models please.

**Comment by Cavcode** [comment]
Models compiled.

File: swlor2_2da/genericdoors.2da [hunk 1]
```diff
@@ -1,132 +1,135 @@
...
+120  ctp_curtain06      111919 ctp_curtain_06 1          1            24           111919   
+121  intoff_wide        5349   tfb_udoor_01   1          1            0            63750    
+122  intoff_wide        5349   tfb_udoor_02   1          1            0            63750    
+123  fence              5349   tfb_udoor_03   0          1            0            63750    
+124  alien_door1        111925 vmr_gdoor01    1          1            4            111925   
+124  alien_door2        111925 vmr_gdoor02    1          1            4            111925   
+124  alien_door3        111925 vmr_gdoor03    1          1            4            111925   
+1361 Jabba_palace_door  5349   ttd_door009    1          1            0            14445    
+1362 Astroport_door     5349   ttd_door_010   1          1            0            14445    
+26   sen_gendoor01      5349   sen_sdoor01    1          1            0            63750
```
Comment (zunath) [inline swlor2_2da/genericdoors.2da:133]: These IDs are wrong.

File: swlor2_2da/doortypes.2da [hunk 2]
```diff
@@ -1,1442 +1,1447 @@
...
+658  ****              ****         ****    ****           ****          ****       ****         ****         
+659  ****              ****         ****    ****           ****          ****       ****         ****         
+660  ****              ****         ****    ****           ****          ****       ****         ****         
+661  ****              ****         ****    ****           ****          ****       ****         ****         
+662  ****              ****         ****    ****           ****          ****       ****         ****         
+663  ****              ****         ****    ****           ****          ****       ****         ****         
+664  ****              ****         ****    ****           ****          ****       ****         ****         
+665  ****              ****         ****    ****           ****          ****       ****         ****         
+666  ****              ****         ****    ****           ****          ****       ****         ****         
+667  ****              ****         ****    ****           ****          ****       ****
```
Comment (zunath) [inline swlor2_2da/doortypes.2da:1443]: These IDs don't line up.

**Comment by zunath** [comment]
This PR will remain on hold until someone makes an area using the tileset. I don't want to bloat the haks with content no one is actually using.

**Comment by zunath** [comment]
Content was never used. Closing the PR. Will reopen if someone decides to make something with it.

**Pull request closed without merging**